### PR TITLE
Updated links to JT-VAE repo as it was renamed

### PR DESCRIPTION
**Pull request by Strandgaard96**

### CODE REVIEW

File: DETAILS.md [hunk 1]
```diff
@@ -9,65 +9,66 @@ In other words, the Python script structures the general process for both the mo
...
-     - `tmcinvdes/structure_generation/assemble_tmcs.py` *(Re-used from $\mathbf{(5 \rightarrow 6)}$.)*
-     - `bash/assemble_tmcs-monodentate-cond_mono-sampled_optimized.sh`
- - $\mathbf{(12 \rightarrow 13)}$: Label TMCs by ORCA calculations on TMCs. The ORCA calculations are done on HPC, the present Python file only parses the ORCA output files. (New Bash files but re-used Python script.)
-     - `tmcinvdes/quantum_chemistry/orca/parse_orca_to_labels.py` *(Re-used from $\mathbf{(6 \rightarrow 7)}$.)*
-     - `bash/parse_orca_to_labels-monodentate-cond_mono-sampled_optimized.sh`
+- $\\mathbf{(0 \\rightarrow 1)}$: Select initial ligands and get their encoded SMILES representation.
+  - `tmcinvdes/ligand_generation/get_encoded_smiles.py`
+  - `bash/get_encoded_smiles-monodentate.sh`
+  - `bash/get_encoded_smiles-bidentate.sh`
+- $\\mathbf{(1 \\rightarrow 2)}$: Train unconditional JT-VAE models. This is done with code in the separate JT-VAE repository [https://github.com/Strandgaard96/JT-VAE-tmcinvdes/](https://github.com/Strandgaard96/JT-VAE-tmcinvdes/) applied to the `.txt` files in $1$.
```
Comment (tlinjordet) [inline DETAILS.md]: We can probably change this instance on this line to `[JT-VAE-tmcinvdes](https://github.com/Strandgaard96/JT-VAE-tmcinvdes/)` as well, i.e., no need to show the entire URL.

We can also use `[JT-VAE-tmcinvdes](https://github.com/Strandgaard96/JT-VAE-tmcinvdes/)` everywhere and not just `[JT-VAE](https://github.com/Strandgaard96/JT-VAE-tmcinvdes/)`.

File: DETAILS.md [hunk 2]
```diff
@@ -9,65 +9,66 @@ In other words, the Python script structures the general process for both the mo
...
+- $\\mathbf{(8b \\rightarrow 9)}$: Train conditional JT-VAE models with ligands labeled by their TMCs' DFT properties. See our [JT-VAE](https://github.com/Strandgaard96/JT-VAE-tmcinvdes/) repository.
+- $\\mathbf{(8b \\rightarrow 10)}$: Sample ligands for optimization.
+  - `tmcinvdes/ligand_generation/sample_ligands_to_optimize.py`
+  - `bash/sample_ligands_to_optimize-monodentate.sh`
+- $\\mathbf{(10 \\rightarrow 11)}$: Optimize sampled ligands. See our [JT-VAE](https://github.com/Strandgaard96/JT-VAE-tmcinvdes/) repository.
+- $\\mathbf{(11 \\rightarrow 12)}$: Assemble TMC structures as XYZ blocks in concatenated output file. (New Bash files but re-used Python script.)
+  - `tmcinvdes/structure_generation/assemble_tmcs.py` *(Re-used from $\\mathbf{(5 \\rightarrow 6)}$.)*
+  - `bash/assemble_tmcs-monodentate-cond_mono-sampled_optimized.sh`
+- $\\mathbf{(12 \\rightarrow 13)}$: Label TMCs by ORCA calculations on TMCs. The ORCA calculations are done on HPC, the present Python file only parses the ORCA output files. (New Bash files but re-used Python script.)
+  - `tmcinvdes/quantum_chemistry/orca/parse_orca_to_labels.py` *(Re-used from $\\mathbf{(6 \\rightarrow 7)}$.)*
```
Comment (tlinjordet) [inline DETAILS.md]: I see now that the italics break the math rendering on the GitHub website. Could you change this line to:
```
  - `tmcinvdes/quantum_chemistry/orca/parse_orca_to_labels.py` *(Re-used from * $\mathbf{(6 \rightarrow 7)}$ *.)*
```
rendering as: 
  - `tmcinvdes/quantum_chemistry/orca/parse_orca_to_labels.py` *(Re-used from* $\mathbf{(6 \rightarrow 7)}$*.)*

**Comment by tlinjordet** [comment]
Thanks for the PR, this is a good size for something without a predefined issue.

I like the new JT-VAE-tmcinvdes repo name!

However, I cannot see the point in adding double backslashes to the math notation, at least in VScode markdown preview it breaks the math rendering. 

Here in GitHub it looks like both versions render as intended:

Current version: `$\mathbf{(1 \rightarrow 2)}$` which renders as $\mathbf{(1 \rightarrow 2)}$; versus
This PR: `$\\mathbf{(1 \\rightarrow 2)}$` which renders as $\\mathbf{(1 \\rightarrow 2)}$.

**Comment by Strandgaard96** [comment]
Somehow the auto-formatting in my markdown editor added the extra backslash. 
Agree, should not be there, is removed now :)

**Review by tlinjordet — CHANGES_REQUESTED** [review]
I had requested a few minor changes in the comments in addition to the double backslash. This may have been missed, hence changing review status to request changes.

**Comment by Strandgaard96** [comment]
Thanks for checking, the links are updated now, as well as the suggested line : )

**Review by tlinjordet — APPROVED** [review]

**Pull request merged**